### PR TITLE
Added documentation for the authz module

### PR DESCRIPTION
**Pull request by croyyds**

### CODE REVIEW

**Review by lezzokafka — COMMENTED** [review]
Thanks @croyyds! Left some small suggestion, other wise looks great.

File: docs/chain-details/module_authz.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+A _grantee_ can `execute` an authorization already granted by the _granter_.
+
+### `grant`:
+An authorization starts from the _granter_ granting the _grantee_.
+
+#### `tx authz grant [grantee_address] send --spend-limit [amount] --from [granter_address]`- **Grant to send**
+
+The _granter_ may grant a _grantee_ to send tokens on the _granter_'s behalf, where a spend limit should be 
+provided through the `--spend-limit` flag.
+For example, _granter_ may authorize _grantee_ to spend up to `10 CRO`
```
[the remark this answers is not in the document]
Comment (croyyds) [reply docs/chain-details/module_authz.md]: Cheers for the suggestion! Modified.

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**